app/models/series.rb

```diff
@@ -207,13 +207,18 @@ def revised_at
   def bookmarkable_json
     as_json(
       root: false,
-      only: [:title, :summary, :hidden_by_admin, :restricted, :created_at,
-        :complete],
-      methods: [:revised_at, :posted, :tag, :filter_ids, :rating_ids,
+      only: [
```

Comment:
Should I fix this for all 3 models (work, series, external work)?

Reply:
Personal opinion: I disagree with this rule to the point where I'd be in favour of turning off this particular "cop".

I think knowing instantly whether an entity is a symbol or string is useful, especially since coders everywhere still get occasionally bitten by hashes having keys as symbols instead of strings and vice versa. I tend to not notice the significance of whatever is used to frame the array.

Reply:
I don't mind -- what is it %w? for strings because it spares the quotation marks, but I really don't see much value in %i here. I'd leave the code as is and be fine turning off the cop, which I'm pretty sure we've always ignored anyway.